src/client/components/DataGrid/DataGrid.scss

```diff
@@ -0,0 +1,38 @@
+@import 'src/client/style/partials';
+
+.data-grid {
+  display: grid;
+  font-family: $font-family-1;
+  margin: $spacing-s 0;
+  width: 100%;
+
+  .head {
+    --ui-accent: $ui-accent;
+    --ui-accent-light: #c4e7eb;
+    --ui-bg: #f5f5f5;
+    --ui-bg-hover: #ebebeb;
+    color: #555;
+    font-family: "Open Sans", sans-serif;
+    direction: ltr;
+    border-collapse: collapse;
+    font-size: 13px;
+    line-height: 19px;
+    box-sizing: inherit;
+    font-weight: 600;
+    padding: 10px;
+    text-align: left;
+    background-color: rgba(0, 0, 0, .04);
+    border: 1px solid #dadada;
+  }
+
+  > :not(.head) {
+    color: #555;
+    border-collapse: collapse;
+    font-size: 13px;
+    font-weight: 400;
+    white-space: nowrap;
+    background-color: white;
+    border: 1px solid #dadada;
+    border-top: none;
+  }
```

Comment:
this belongs to data-grid-column. and use affermative condition. plus move this css to DataColumn.scss

create a class .data-grid-column {
&.head{}
}